src/einsteinpy/hypersurface/embedding.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import math
```

Comment:
We would like to remain in numpy ecosystem

Reply:
Yes, as pointed out by @shreyasbapat omit `math` and use `np.sqrt` and other `numpy` functions wherever possible.